stdlib/public/core/VarArgs.swift

```diff
@@ -290,7 +290,9 @@ final public class VaListBuilder {
     // supported vararg type is greater than the alignment of Int.
     // FIXME: this implementation is not portable because
     // alignof differs from the ABI alignment on some architectures
-#if os(watchOS) && arch(arm)   // FIXME: rdar://21203036 should be arch(armv7k)
+    // FIXME: This is valid for AAPCS calling convention, FreeBSD/arm
+    // may need this too.
+#if (os(watchOS) || os(Linux)) && arch(arm)   // FIXME: rdar://21203036 should be arch(armv7k)
```

Comment:
Nah, it's more "any other hypothetical platforms using the same ABI as watchOS", i.e. if Apple did another 32-bit ARM platform, it's likely to be more like watchOS than iOS.